### PR TITLE
Feature/spp 6/admin dashboard v0.1a

### PR DESCRIPTION
**Pull request by oldmcdonnell**
added different views, added collapsible component, added search function... added placeholder for map.

### CODE REVIEW

File: test.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { PrismaClient } from '@prisma/client';
+
+const prisma = new PrismaClient();
+
+async function test() {
+    const users = await prisma.user.findMany();
+    console.log(users);
+}
+
+test();
```
Comment (davideastmond) [inline test.ts:10]: Do we need this? Can we delete it?

File: src/lib/utils/search/SearchRequests.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        type="text"
+        placeholder="Search by name or email"
+        className="border border-gray-300 rounded-md px-4 py-2 mr-2 w-full"  //not sure if i need this... 
+        onChange={(e) => handleSearch(e.target.value)}
+      />
+    </div>
+  );
+};
+
+export default SearchRequests;
```
Comment (davideastmond) [inline src/lib/utils/search/SearchRequests.tsx:41]: This looks like a react component - can we move it to the components folder?

File: prisma/seed-scheduled-times.ts [hunk 3]
```diff
@@ -0,0 +1,80 @@
+import { PrismaClient, RequestStatus } from "@prisma/client";
+
```
Comment (davideastmond) [inline prisma/seed-scheduled-times.ts:2]: Opinion: now that we have submission form working - we can create appointments using the app itself. Right now, a user can create multiple requests under one id, and we can use that for testing. 

I am thinking we don't need this seeding code anymore.

File: prisma/seed-users-and-addresses.ts [hunk 4]
```diff
@@ -0,0 +1,121 @@
+import { PrismaClient } from "@prisma/client";
```
Comment (davideastmond) [inline prisma/seed-users-and-addresses.ts:1]: Same here. It's a lot of code. We can create data during normal testing of the app now that we have submission form working.

**Review by davideastmond — COMMENTED** [review]
I just left a few comments. I am open to other opinions, but I don't think we need the seeding code now that the submission form is done.

File: test.ts (hunk 1 above)
Comment (oldmcdonnell) [reply test.ts:10]: yes delete... this is not mine.

File: src/lib/utils/search/SearchRequests.tsx (hunk 2 above)
Comment (oldmcdonnell) [reply src/lib/utils/search/SearchRequests.tsx:41]: Yes.. i did ask btw

File: prisma/seed-scheduled-times.ts (hunk 3 above)
Comment (oldmcdonnell) [reply prisma/seed-scheduled-times.ts:2]: i did not update that file. that must be a vestige

**Pull request closed without merging**